cmd/cmd_bucket.go

```diff
@@ -4,9 +4,10 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"time"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
```

Comment:
delete this blank line.

Reply:
The 15th `github.com/urfave/cli/v2` should be sorted.

Reply:
fixed